Makefile

```diff
@@ -83,4 +104,9 @@ create-install-files-dev:
 	IMAGE_NAME_NODE_AGENT=$(NS)/$(IMAGE_NAME_NODE_AGENT) \
 	INIT_CONTAINER_TAG=$${INIT_CONTAINER_TAG} \
 	NODE_AGENT_TAG=$${NODE_AGENT_TAG} \
-	make -C operator create-install-files
\ No newline at end of file
+	make -C operator create-install-files

```

Comment:
this should be pkg/operator?

Reply:
both for create-install-files and create-install-files-dev steps

Reply:
```
dgkanatsios@Dimitrioss-MacBook-Air thundernetes % make create-install-files-dev
mkdir -p ./installfilesdev && \
        INSTALL_FILES_FOLDER=installfilesdev \
        IMG=ghcr.io/playfab//thundernetes-operator:${OPERATOR_TAG} \
        IMAGE_NAME_INIT_CONTAINER=ghcr.io/playfab//thundernetes-initcontainer \
        IMAGE_NAME_NODE_AGENT=ghcr.io/playfab//thundernetes-nodeagent \
        INIT_CONTAINER_TAG=${INIT_CONTAINER_TAG} \
        NODE_AGENT_TAG=${NODE_AGENT_TAG} \
        make -C pkg/operator create-install-files
cd config/manager && /Users/dgkanatsios/Desktop/projects/thundernetes/pkg/operator/bin/kustomize edit set image controller=ghcr.io/playfab//thundernetes-operator:
/Users/dgkanatsios/Desktop/projects/thundernetes/pkg/operator/bin/kustomize build config/default | API_SERVICE_SECURITY=none envsubst > ../installfilesdev/operator.yaml
bash: ../installfilesdev/operator.yaml: No such file or directory
make[1]: *** [create-install-files] Error 1
make: *** [create-install-files-dev] Error 2
```
checking

Reply:
oh, these steps on the pkg/operator/Makefile need to be edited as wel
```$(KUSTOMIZE) build config/default | API_SERVICE_SECURITY=none envsubst > ../${INSTALL_FILES_FOLDER}/operator.yaml``` 
operator.yaml is one folder up on the new layout

Reply:
running NS=testcr.io/dgkanatsios/ make create-install-files-dev produces correct result for init container but `latest` for controller, plus there is extra '/' for both

```
  value: testcr.io/dgkanatsios//thundernetes-initcontainer:02f0fb6
        - name: TLS_SECRET_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        image: testcr.io/dgkanatsios//thundernetes-operator:latest
```

Reply:
running `make create-install-files` produces correct tag for controller but wrong one for init container and node agent
```
- name: THUNDERNETES_INIT_CONTAINER_IMAGE
          value: ghcr.io/playfab//thundernetes-initcontainer:02f0fb6
        - name: TLS_SECRET_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        image: ghcr.io/playfab//thundernetes-operator:0.1.0
```

Reply:
should be corrected with the latest update!